common/script/ops/scoreTask.js

```diff
@@ -90,79 +109,80 @@ function _gainMP (user, val) {
   }
 }
 
-// HP modifier
-// ===== CONSTITUTION =====
-// TODO Decreases HP loss from bad habits / missed dailies by 0.5% per point.
+// Decrease user's health when:
+// - user does negative Habits
+// - cron finds uncompleted Dailies
 function _subtractPoints (user, task, stats, delta) {
-  let conBonus = 1 - user._statsComputed.con / 250;
+  let conBonus = 1 - user._statsComputed.con / 250; // CON decreases HP loss
```

Comment:
Make this `* 0.004` instead.  As-is, it doesn't fit with the way `intBonus` and `perBonus` are calculated (below).